apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -147,6 +149,14 @@ class SectionProgress extends Component {
       (currentView === ViewType.SUMMARY || currentView === ViewType.DETAIL);
     const standardsStyle =
       currentView === ViewType.STANDARDS ? styles.show : styles.hide;
+
+    // Reorder coursesWithProgress so that the current section is at the top and other sections are in order from newest to oldest
+    // Also adds "(Current Section)" label to the course version in the currently viewed section.
+    const reorderedCourses = [
```

Comment:
I still think this isn't the best place to do the sorting because the teacher will still see the first course in the originally-ordered list when they load the page. I did a quick test locally and if you put the sorting [here](https://github.com/code-dot-org/code-dot-org/blob/9b382827296eb3fde84380c0a210fa1d5f6fa043/apps/src/sites/studio/pages/teacher_dashboard/show.js#L83) instead the assigned course is shown on page load.

Reply:
Good point. I need to brush up on my redux knowledge.